common/pkgfiles/referencedfilelist.cpp

```diff
@@ -1018,6 +999,62 @@ bool ReferencedFileList::addFilesFromQuery(IConstWorkUnit *cw, const IHpccPackag
                 ensureFile(logicalName, flags, NULL, false, &subfileNames);
         }
     }
+    else
+    {
+        Owned<IConstWUGraphIterator> graphs = &cw->getGraphs(GraphTypeActivities);
+        ForEach(*graphs)
+        {
+            Owned <IPropertyTree> xgmml = graphs->query().getXGMMLTree(false, false);
+            Owned<IPropertyTreeIterator> iter = xgmml->getElements("//node[att/@name='_*ileName']");
```

Comment:
'_*ileName' looks odd, is it because it can be _fileName, or spillFileName or similar? could do with a comment.

Reply:
(but I now see this is just moved code, not new code)